src/interface/chat/thinking.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Box, Text } from 'ink';
+import { Spinner } from '@inkjs/ui';
+
+type ThinkingProps = {
+  thinking: boolean;
+};
+
+export const Thinking = ({ thinking }: ThinkingProps) => {
```

Comment:
[nit] ThinkingMessage

Reply:
MessagePlaceholder fit the rest of the code :P